docs/source/development/eep-00-governance-model.md

```diff
@@ -0,0 +1,159 @@
+(eep-00)=
+
+
+# EEP-00: Governance model & code of conduct
+
+```{eval-rst}
++------------+------------------------------------------------------------------+
+| Author     | `Maximilian Blesch <https://github.com/MaxBlesch>`_,             |
+|            | `Janoś Gabler <https://github.com/janosg>`_,                     |
+|            | `Hans-Martin von Gaudecker <https://github.com/hmgaudecker>`_,   |
+|            | `Annica Gehlen <https://github.com/amageh>`_,                    |
+|            | `Sebastian Gsell <https://github.com/segsell>`_,                 |
+|            | `Tim Mensinger <https://github.com/timmens>_,                    |
+|            | `Mariam Petrosyan <https://github.com/mpetrosian>`_,             |
+|            | `Tobias Raabe <https://github.com/tobiasraabe>`_,                |
+|            | `Klara Röhrl <https://github.com/roecla>`_                       |
++------------+------------------------------------------------------------------+
+| Status     | Draft                                                            |
```

Comment:
Wouldn't we need to change the status here once adopted?

Reply:
Done.